README.md

```diff
@@ -102,3 +102,44 @@ end
 iex> person(123, "Josh", "Nussbaum") |> generate
 "<person id=\"123\"><first>Josh</first><last>Nussbaum</last></person>"
 ```
+
+### DOCTYPE declarations
+
+A DOCTYPE can be declared by applying the `doctype` function at the first position of a list of elements in a `doc` definition:
+
+```elixir
+import XmlBuilder
+
+doc([
+  doctype("greeting", system: "hello.dtd"),
+  element(:person, "Josh")
+])
+```
+
+Outputs
+
+```xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE greeting SYSTEM "hello.dtd">
+<person>Josh</person>
+```
+
+Alternatively with a public identifier:
```

Comment:
How do you feel about having only one doctype example in the readme?
and we can move the second example to the docs for the `doctype` function